app/server/methods/data.js

```diff
@@ -1,64 +1,22 @@
-const async = require("async");
-
-async function _exportDataToConcernedColection(collectionName, data) {
+function _exportDataToConcernedColection(collectionName, data) {
   if (!Array.isArray(data)) throw "Data not in required format";
 
-  console.log("Calling data for coll: ", collectionName);
-
-  switch (collectionName) {
-    case "activities":
-      return await _insert(Activities.insert, data);
-
-    case "activityTypes":
-      return await _insert(ActivityTypes.insert, data);
-
-    case "feelings":
-      return await _insert(Feelings.insert, data);
-
-    case "groups":
-      return await _insert(Groups.insert, data);
-
-    case "homes":
-      return await _insert(Homes.insert, data);
-
-    case "residents":
-      return await _insert(Residents.insert, data);
-
-    case "roles":
-      return await _insert(Meteor.roles.insert, data);
-
-    case "settings":
-      return await _insert(Settings.insert, data);
-
-    case "users":
-      return await _insert(Meteor.users.insert, data);
-  }
-  function _insert(func, data) {
-    // let insertMethod = Meteor.wrapAsync(func);
-
-    return new Promise((resolve, reject) => {
-      async.each(
-        data,
-        Meteor.bindEnvironment((row, cb) => {
-          func(
-            row,
-            Meteor.bindEnvironment((err, res) => {
-              if (err) cb(err);
-              else cb();
-            })
-          );
-          /* data.forEach(row => {
-      insertMethod(row);
-    }); */
-        }),
-        Meteor.bindEnvironment(err => {
-          if (err) return reject(err);
-
-          return resolve("Success");
-        })
-      );
-    });
-  }
+  let collections = {
+    activities: Activities,
+    activityTypes: ActivityTypes,
+    feelings: Feelings,
+    groups: Groups,
+    homes: Homes,
+    residents: Residents,
+    residencies: Residencies,
+    roles: Meteor.roles,
+    settings: Settings,
+    users: Meteor.users
+  };
+  return data.every(function(row) {
+    let res = collections[collectionName].insert(row);
```

Comment:
Use `const` here, since the reference will not change. In general, always use `const`, unless the reference will change.